index.php

```diff
@@ -0,0 +1,69 @@
+<?php
+session_start();
+require_once('database/db.php');
+require_once('security/connexion.php');
```

Comment:
Replace "require_once" with namespace import mechanism through the "use" keyword.